src/app/views/find/overview/overview.component.ts

```
@@ -0,0 +1,25 @@
+@Component({
+  selector: 'app-overview',
+  imports: [AveragesTableComponent, ToolbarComponent, CommonModule],
+  templateUrl: './overview.component.html',
+  styleUrls: ['./overview.component.scss']
+})
+export class OverviewComponent implements OnInit {
+  showContainer = inject(BreakpointService).showContainer;
+  mockDataSvc = inject(MockDataService);
+  exerciseAverages = toSignal(this.mockDataSvc.getExerciseAverages());
+
+  constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)